gruntfile.js

```diff
@@ -157,4 +162,5 @@ module.exports = function(grunt) {
     grunt.registerTask('default', ['install', 'versionupdater', 'compress']);
     grunt.registerTask('version', ['versionupdater']);
     grunt.registerTask('zip', ['compress']);
+    grunt.registerTask('lint_js', 'Check JS syntax', ['eslint']);
```

Comment:
Few lines above, we use `client:install`. Not sure if it is recommended practice but we should probably be consistent.